CONTRIBUTING.md

```diff
@@ -41,6 +41,23 @@ which is a test framework originally designed for
 We invite and recommend developers to write new integration tests
 using this framework.
 
+**Adding a non-regression test on an already existing command:**
+
+1. Identify which module of [`tests/dedukti.mli`](tests/dedukti.mli)
```

Comment:
You could add something about that if the command does not exist, it should be added